src/twitter-preview/twitter-preview.js

```diff
@@ -15,6 +15,17 @@
     type
   })
 
+  /**
+   * why: inside iframe we have no access to custom properties of parent
+   * how: get needed props based on their key
+   * possible: extend 'propKeys' with additional props
+   */
+  const propKeys = ['--base-color', '--label-color', '--value-color', '--divider-border', '--toolbar-bg-color'];
```

Comment:
I see there's a separate PR #13 for the dark mode Twitter preview. So I'd suggest keeping this part out of this PR.

(I actually think the `twitter-preview` should be standalone, so only receive a `?theme=dark` parameter and figure out its dark mode styling based on that of mobile.twitter.com rather than using the colors from Chrome Devtools)